fearless/Modules/Staking/SelectValidators/ViewModel/SelectValidatorsViewModelFactory.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+import FearlessUtils
+
+final class SelectValidatorsViewModelFactory: SelectValidatorsViewModelFactoryProtocol {
+    private lazy var iconGenerator = PolkadotIconGenerator()
+
+    func createViewModel(validators: [ElectedValidatorInfo]) -> [SelectValidatorsCellViewModel] {
+        let percentageAPYFormatter = NumberFormatter.percent.localizableResource()
+        return validators.map { validator in
+            let icon = try? self.iconGenerator.generateFromAddress(validator.address)
+            let restakePercentage = percentageAPYFormatter
+                .value(for: .current) // TODO: return LocalizebleRes<[SelectValidatorsCellViewModel]>
```

Comment:
Please, take a look to crowdloans implementation in terms of localization. It might be a good idea to track localization in also in presenter to get rid of Localizable boilerplate.

Reply:
Sure. This piece of code is from the next feature, I'll consider what you said in development process